### PR TITLE
Don't try to handle ties in a County context

### PR DESCRIPTION
**Pull request by sduckett**
Now that we have contests spanning counties, we need to wait until all possible CVR exports have been uploaded before we can count the results and look for any ties.

By doing it at the time of CVR import, a contest that might have tied in a county - but not overall - gets placed in the DoSDashboards.contestsToAudit() when it has no business being there yet.

### CODE REVIEW

File: server/eclipse-project/src/main/java/us/freeandfair/corla/model/DoSDashboard.java [hunk 1]
```diff
@@ -263,16 +263,6 @@ public boolean updateContestToAudit(final ContestToAudit the_contest_to_audit) {
...
 
-  /**
-   * data access helper
-   * @return contests of contestsToAudit a.k.a selected contests, targeted contests
-   */
-  // FIXME this might not be needed.
-  public Stream<String> targetedContestNames() {
-    return contestsToAudit().stream()
-      .map(a -> a.contest().name());
-  }
```
Comment (gws) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/model/DoSDashboard.java]: Totally unused before, or removed a call site in this PR?

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRExportImport.java [hunk 2]
```diff
@@ -615,75 +614,4 @@ private int cleanup(final County the_county, final boolean the_failure_flag,
...
-      if (success && retries > 1) {
-        Main.LOGGER.info("updated DoS dashboard for county " + the_county.id() + 
-                         " tied contests in " + retries + TRIES);
-      } else if (!success) {
-        error("could not update DoS dashboard for county " + the_county.id() + 
-              " tied contests after " + retries + TRIES);
-      } 
-      return result;
-    }
-  }
```
Comment (gws) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRExportImport.java]: I think you removed one too many `}`

File: server/eclipse-project/src/main/java/us/freeandfair/corla/model/DoSDashboard.java (hunk 1 above)
Comment (sduckett) [reply server/eclipse-project/src/main/java/us/freeandfair/corla/model/DoSDashboard.java]: Totally unused.

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRExportImport.java (hunk 2 above)
Comment (sduckett) [reply server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRExportImport.java]: I think I was trying to make a diff that didn't have a handful of whitespace changes, good catch.

**Pull request merged**